pytentiostat/config_reader.py

```diff
@@ -8,22 +8,21 @@
 def parse_config_file(configlocation=None):
     """
     Reads config data from the config file
-
```

Comment:
This line should be put back in.  First line is a one-line (<80 chars) description of the overall purpose of the function,  then a blank line, then as many lines as you want to expand on the explanation, then a blank line, then Parameters underlined and so on.

Reply:
Okay, makes sense!